orcoursetrion/actions/github.py

```diff
@@ -179,6 +209,35 @@ def rerun_xml(course, term):
     return github.delete_web_hooks(config.ORC_XML_ORG, repo_name)
 
 
+def release_xml(course, term):
+    """Moves an XML course to be ready for production.
+
+    Currently this will just add a hook to the production server, but
+    it will eventually take care of everything else needed for a
+    transfer as well (i.e. making live branch, import it to lms...).
+
+    Args:
+        course (str): Course name of the repo to release to production
+                      (i.e. 6.001)
+        term (str): Term the course is currently running in (i.e. 2015_Spring)
+    Raises:
+        requests.RequestException
+        orcoursetrion.lib.GitHubUnknownError
+    Returns:
+        None: Nothing returned, raises on failure
+    """
+    github = GitHub(config.ORC_GH_API_URL, config.ORC_GH_OAUTH2_TOKEN)
+    repo_name = '{prefix}-{course}-{term}'.format(
+        prefix=config.ORC_COURSE_PREFIX,
+        course=course.replace('.', ''),
+        term=term
+    )
+    # Add the hook
+    github.add_web_hook(
+        config.ORC_XML_ORG, repo_name, config.ORC_PRODUCTION_GITRELOAD
+    )
```

Comment:
This method seems identical to the one above except for the org name. Do you see these methods diverging in the future? And even if so, should we still have a common path for adding these hooks?
The difference of factoring it out might be negligible after #31, though.